### ISSUE TITLE
The performance of AEC training is collapsed

### ISSUE THREAD

**Issue by YihengJiang** [Xiaobin-Rong/deepvqe#14]
Hello author:
    I hope you're doing well! I've been using your code to train an AEC model, but the results on the AEC-Challenge2023 blind test set aren't looking great — they differ quite a bit from the performance in original DeepVQE paper. Have you trained specifically for the AEC task before? If so, and if you had success, would you be able to share the training code? Any advice on important considerations for working with the AEC-Challenge2023 dataset would also be greatly appreciated.

Thank you very much for your help!

**Comment by Xiaobin-Rong** [comment]
Hi, as stated in the README, I haven't trained the model on the AEC task.
